### ISSUE TITLE
Make it embedable

### ISSUE THREAD

**Issue by jankocian** [yournextstore/yournextstore#43]
Hi all, 
great idea, awesome job – I can easily imagine using this for small e-shops.

I have a challenging feature request – it would be super cool to be able to embed the e-shop via client side JS into an existing (even statically built or Webflow) site, as an alternative to the default Stripe Buy now/Pricing tables. 

I'd love that feature so much! 

Keep it up ;)
J

**Comment by zaiste** [comment]
Interesting idea! For starters have you tried using `<iframe>`?

**Comment by jankocian** [comment]
Not really, I don't think iframed e-commerce store would provide good UX – yet I might be overlooking some options...

**Comment by typeofweb** [comment]
The most popular Checkout in the world uses iframes – Stripe Checkout :)

We don't have that on our roadmap, but it's definitely something worth looking into. Like a widget that you just drop on your website. I like the idea.
